src/app/shared/testing/SidebarServiceStub.ts

```diff
@@ -0,0 +1,14 @@
+import {of as observableOf } from 'rxjs';
+
+export class SidebarServiceStub {
```

Comment:
Tiny thing, could we name this file `sidebar-service.stub.ts`?  That'd keep with the current naming convention, as I see we have other stubs named `*.stub.ts` in this same directory.

Reply:
done